src/ctapipe/image/muon/intensity_fitter.py

```diff
@@ -42,7 +42,12 @@
 @vectorize([double(double, double, double)], cache=True)
 def chord_length(radius, rho, phi):
     """
-    Function for integrating the length of a chord across a circle (effective chord length)
+    Function for integrating the length of a chord across a circle (effective chord length).
```

Comment:
Add a blank line between one-line summary and the body of the docstring. Add the paper to [references](https://github.com/cta-observatory/ctapipe/blob/main/docs/references.bib) if it is not yet there, and cite it similar to this: https://github.com/cta-observatory/ctapipe/blob/26ded3fc2e411f93fa5d1ab6f38ec3ddb86652f9/src/ctapipe/image/cleaning.py#L378C5-L380C73